Specification.md

```diff
@@ -55,21 +66,30 @@ A *should-total* MUST be a *duration* value
 followed by a `!`
 and wrapped in “parentheses”.
 
-### Summary
-A *summary* is user-provided text for holding arbitrary information.
-
-There are two places where *summary* text MAY appear in *records*:
-- Underneath the *date*:
-  In this case the *summary* is considered to be associated with the entire *record*.
-  The *summary* MAY span multiple lines.
-  Each of its lines MUST NOT start with “blank characters”.
-- Behind *entries*:
-  In this case the *summary* is only considered to be referring to the corresponding *entry*.
-  The *summary* text follows the *entry* on the same line,
-  and it ends at the end of that line.
-  It MUST be separated from the *entry* by one “space”.
-
-### Tags
+### Summaries
+A *summary* is user-provided text for capturing arbitrary information
+about a *record* or an *entry*.
+
+#### Record Summary
+The *record summary* MAY appear underneath the *date*.
+It is considered to be associated with the entire *record*.
+
+The *record summary* MAY span multiple lines.
+Each of its lines MUST NOT start with “blank characters”.
+
+#### Entry Summary
+An *entry summary* MAY appear behind an *entry*.
```

Comment:
I think "An" should be "The". It is more consistent with the other paragraphs, as well with the "Record Summary" section. Although there would usually be more than 1 entry summary per record, the spec is only concerned with a single entry summary.

The wording "appear behind" is a little bit ambiguous. Even with the wording on line 88-89, I still read this as appearing to the left of the entry (my interpretation though). Maybe clarify this?

Reply:
> Although there would usually be more than 1 entry summary per record, the spec is only concerned with a single entry summary.

Not sure I understand you correctly – there can be x entries per record, and each record can have one or none entry summary. (Which may span multiple lines, though.)

> The wording "appear behind" is a little bit ambiguous. Even with the wording on line 88-89, I still read this as appearing to the left of the entry (my interpretation though). Maybe clarify this?

Yeah, I wasn’t happy with that either. I’ve rephrased it now. It’s also more in line with the other sections, where the first sentence is a high-level explanation of what something is, and the rest is the actual specification.

I’ve also made it more explicit that summaries are optional.